src/layouts/secondary-navigation-layout/side-nav/SideNavHeader.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Button, { ButtonVariants } from 'terra-button';
+import IconLeft from 'terra-icon/lib/icon/IconLeft';
+import { ThemeContext } from '@cerner/terra-application/lib/theme';
+
+import styles from './SideNavHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * A function called to request close of the side nav.
+   */
+  onRequestClose: PropTypes.function,
+
+  /**
+   * The label for the side nav header.
+   */
+  label: PropTypes.string,
+};
+
+const SideNavHeader = ({
+  onRequestClose, label,
+}) => {
+  const theme = React.useContext(ThemeContext);
+  return (
+    <div className={cx('page-header-container', theme.className)}>
+      <div className={cx('page-layout-header')}>
+        {onRequestClose ? (
+          <div className={cx('back-button-container')}>
+            <Button
+              className={cx(['header-button', 'back-button'])}
+              icon={<IconLeft />}
+              text="Back" // TODO validate icon/text and intl
```

Comment:
We could lift one of the ActionHeader translations here (Back or Close I dunno which)

Reply:
I went with back.